src/renderer/viz/expressions/top.js

```diff
@@ -44,8 +44,8 @@ export default class Top extends BaseExpression {
 
         let ret;
         orderedCategoryNames.map((name, i) => {
-            if (i === p) {
-                ret = i < buckets ? i + 1 : 0;
+            if (i === catID) {
+                ret = i < buckets ? this._meta.IDToCategory.get(i) : 'CARTOVL_TOP_OTHERS_BUCKET';
```

Comment:
Where is this used `CARTOVL_TOP_OTHERS_BUCKET`?

Reply:
It's part of the new public API, see https://github.com/CartoDB/carto-vl/pull/782/files#diff-b603c20d9a415dae327ff3eb208e6794R67

Reply:
OK, so this name is just to avoid collision with the categories

Reply:
Yes, exactly

Reply:
Using a Symbol (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol) would be the right thing to do, but the browser support is not super good yet